CMakeLists.txt

```diff
@@ -97,7 +97,8 @@ message(STATUS "Csound directory: ${CMAKE_HOME_DIRECTORY}")
 
 
 # Project definitions
-set(APIVERSION "7.0")
+set(CS_APIVERSION "7")
```

Comment:
I'm a bit concerned we have API version vs. the regular version, subver, patchlevel. With semantic versioning, we should really just use the latter and remove APIVERSION IMO. App developers should be able to depend on the major, minor, patch to handle backwards compatibility.

Reply:
I'm happy with this. I can remove the API version reporting from the host API and we're done with it.